cockroachcloud/configure-scim-provisioning.md

```diff
@@ -0,0 +1,118 @@
+    - **API authentication token**: the API token for a {{ site.data.products.db }} [service account](managing-access.html#create-a-service-account) with the [**Org Administrator (legacy)** role](authorization.html#org-administrator-legacy).
+    - **Unique identifier field for users**: `userName`
+    - **Authentication Mode**: **HTTP Header**
+
+1. Click **Test Connector Configuration**.
+1. Click **Save**.
+1. Click **To App**. This tab controls assignment of Okta identities to {{ site.data.products.db }}. To allow provisioning and deprovisioning of users, ensure that **Create Users** and **Deactivate Users** are selected, and make any other desired changes.
+1. Optionally, click **To Okta**. This tab allows you to perform a one-time import of a {{ site.data.products.db }} organization's existing users into Okta. Refer to Okta's documentation about mapping individual fields. Make any desired changes.
+
+To learn more, refer to [Add SCIM Provisioning to App Integrations](https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_SCIM.htm) in the Okta documentation
+
+## Assign a user
+
+To provision an Okta user to {{ site.data.products.db }}, assign the user to the SAML application.
+
+1. Log in to Okta Admin Dashboard as an admin user.
+1. Click **Applications** and click the [SAML](configure-cloud-org-sso.html#saml) application for your {{ site.data.products.db }} organization.
+1. Click **Assignments**. When an IdP user or group is assigned to an application, Okta allows them to sign in to the application.
+1. Click **Assign**, then select **Assign to People** or **Assign to Groups**.
+
+      Operations on an IdP group are applied to each member of the group individually by the {{ site.data.products.db }} API. For example, assigning an IdP group to the SCIM application allows the group's members to sign in to the application.
+
+1. Filter or search for a user or group.
+1. Next to them, click **Assign**, then **Save and go back**.
+
+      {{ site.data.products.db }} accounts are provisioned for IdP users who are assigned to the SAML application directly. If an IdP user is assigned to an application indirectly via membership in an IdP group, an account is provisioned only when the user first successfully authenticates to {{ site.data.products.db }} using the SAML application or when an IdP admin clicks **Push Groups**..
+
+1. Instruct the user how to access your {{ site.data.products.db }} organization. {{ site.data.products.db }} does not notify a user when an account is provisioned for them using SCIM. Users may use your IdP's web interface or a browser plugin, or they may access your {{ site.data.products.db }} organization's custom login URL directly and select an SSO login method.
+
+To learn more, refer to [Assign An App Integration to a User](https://help.okta.com/en-us/Content/Topics/Provisioning/lcm/lcm-assign-app-user.htm) in the Okta documentation.
+
+## Unassign a user
```

Comment:
We should probably make a note in this section that Okta does not support deleting users, so unassignments results in disabled users instead.